### PR TITLE
Fix rollbar "No instance family members for rightsizing."

### PR DESCRIPTION
**Pull request by eb1909**
Add missing instance types to classification data structure, and add a test to make sure all instance types we know about are represented

### CODE REVIEW

File: spec/lib/amazon-pricing/helpers/instance-type-spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+describe 'AwsPricing::Helper::InstanceType' do
+  let(:dummy_class) { Class.new { include AwsPricing::Helper::InstanceType } }
+  let(:helper) {dummy_class.new }
+
+  context 'family_members' do
+
+    # get a list of supported instance types, and filter out anything with more than 2 parts to its name (to get
+    # rif of alias-y things like cache.m3.large and db.m3.large)
+    types = AwsPricing::Ec2InstanceType.instance_eval("@Network_Performance").keys.reject {|k| k =~ /.*\..*\..*/}
+    puts types
```
Comment (aschepis) [inline spec/lib/amazon-pricing/helpers/instance-type-spec.rb]: remove puts

File: lib/amazon-pricing/helpers/instance-type.rb [hunk 2]
```diff
@@ -139,6 +139,7 @@ def family(api_name)
       end
 
       def family_members(api_name)
+        puts "checking #{api_name}"
```
Comment (aschepis) [inline lib/amazon-pricing/helpers/instance-type.rb]: remove puts

**Comment by eb1909** [comment]
Removed the debug output.

**Comment by aschepis** [comment]
👍

**Comment by lardcanoe** [comment]
:+1:

**Pull request merged**